lang_qc/models/pacbio_well.py

```diff
@@ -21,31 +21,25 @@
 
 from pydantic import BaseModel, Field
 
+from lang_qc.models.qc_state import QcStatus
 
-class Study(BaseModel):
 
-    id: str = Field(default=None, title="Study ID.")
+class WellInfo(BaseModel):
 
+    label: str = Field(
+        default=None, title="Well label", description="The well identifier."
+    )
+    start: datetime = Field(default=None, title="Timestamp of well started")
+    complete: datetime = Field(default=None, title="Timestamp of well complete")
+    qc_status: QcStatus = Field(default=None, title="Well QC status")
 
-class Sample(BaseModel):
-
-    id: str = Field(default=None, title="Sample ID")
 
+class InboxResultEntry(BaseModel):
```

Comment:
I think the "result" here is unnecessary, otherwise we would need result in any chunk of data we served from the API. Agree?

Reply:
The classes in this package will change in the next pr, I'll certainly change this class name